src/tests.rs

```diff
@@ -0,0 +1,88 @@
+use crate::{self as pallet_template, Config };
```

Comment:
Don't call your pallet `pallet_template` it's not a template, it's the `pallet_subscription`

Reply:
ok